File: python/handlers.py
class Handlers:
    @staticmethod
    def ping(command):
        return "PONG"

    @staticmethod
    def status(command):
        return "STATUS:OK"
    
    @staticmethod
    def uptime(command):
        return "UPTIME:0"
    
    @staticmethod
    def echo(command):
        return command.payload

File: java/src/main/java/Handlers.java
public class Handlers {
    public static String ping(Command command) {
        return "PONG";
    }

    public static String status(Command command) {
        return "STATUS:OK";
    }

    public static String uptime(Command command) {
        return "UPTIME:0";
    }

        public static String echo(Command command) {
        return command.getPayload();
    }
}
